CODEOWNERS

```diff
@@ -1,13 +1,13 @@
 * @Microsoft/onnxruntime
 
 # Python frontend owners
-orttraining/*.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
-orttraining/orttraining/python/** @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
-orttraining/orttraining/test/python/** @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
-orttraining/pytorch_frontend_examples/** @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
-onnxruntime/python/training/** @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani @mrry
-onnxruntime/test/python/onnxruntime_test_ort_trainer.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
-onnxruntime/test/python/onnxruntime_test_ort_trainer_with_mixed_precision.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
-onnxruntime/test/python/onnxruntime_test_training_unit_tests.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
-onnxruntime/test/python/onnxruntime_test_training_unittest_utils.py @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
-samples/python/** @thiagocrepaldi @spandantiwari @BowenBao @liqunfu @baijumeswani
+orttraining/*.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+orttraining/orttraining/python/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+orttraining/orttraining/test/python/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+orttraining/pytorch_frontend_examples/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/python/training/** @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_ort_trainer.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_ort_trainer_with_mixed_precision.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_training_unit_tests.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+onnxruntime/test/python/onnxruntime_test_training_unittest_utils.py @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
+samples/python/**@thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad @thiagocrepaldi @tlh20 @BowenBao @liqunfu @baijumeswani @SherlockNoMad
```

Comment:
```suggestion
samples/python/**@thiagocrepaldi @BowenBao @liqunfu @baijumeswani @SherlockNoMad @tlh20
```

Reply:
there is some dupe here

Reply:
please remove the ownership on samples/python/** line or carve out a specific training subdirectory.

Reply:
@SherlockNoMad you could create an ormodule folder inside samples/python to separate from the other components, such as inference or quantization